chapt1.md

```diff
@@ -2,6 +2,8 @@
 
 My suggestion that we think of bible as an anarchic interface has two main aims. First, bible as interface can challenge ideas and practices that reduce bible to simply a container of content to be consumed and can guide our imagination of bible in a time of major media transition. Second, an interface that resists our recourse to an original text or singular governing principle in the act of reading can help focus us on the vast productive potentialities of this interface even as it is translated into media technologies we have yet to imagine. This first chapter will focus on the first of these aims by advocating for a concept^[Might add a note here from Piotr about the preference for concept over definition.] of interface as a space and a process that facilitate relationship without eradicating difference. 
```

Comment:
Though I think the diff between concept and definition is instructive, is it worth doing this work here or should it wait for elsewhere?